code/modules/research/designs/medical_designs.dm

```diff
@@ -665,13 +665,6 @@
 	research_icon_state = "surgery_any"
 	var/surgery
 
-/datum/design/surgery/lobotomy
-	name = "Lobotomy"
-	desc = "An invasive surgical procedure which guarantees removal of almost all brain traumas, but might cause another permanent trauma in return."
-	id = "surgery_lobotomy"
-	surgery = /datum/surgery/advanced/lobotomy
-	research_icon_state = "surgery_head"
```

Comment:
removing normal lobotomies means that abductors can no longer lobotomize people without mugging the psychologist or cargo, and means that you can't speed up the chain lobotomization process as much with alien (or advanced) tools.

Reply:
Time to make an Alien Version of these single-case tools. I don't feel bad about this because of the Chaplain's 50 Null Rods.